src/components/organisms/Modal/useModal.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import Modal from "components/organisms/Modal/Modal";
+
+const useModal = (initialState = false) => {
+  const [isOpen, setModalState] = useState(initialState);
+
+  const handleOpenModal = () => setModalState(true);
+  const handleCloseModal = () => setModalState(false);
```

Comment:
w takich przypadkach lubie sobie zrobić taki zapis:

```
const handleModalOpen = (isOpen: boolean) => () => setModalState(isOpen);
```

i później np jak mamy buttona, to robić coś takiego:
```
<Button onClick={handleModalOpen(true)}>Open</Button>
<Button onClick={handleModalOpen(false)}>Close</Button>
```
troche tricky ale pozwala zachować porządek



bardzo ważne w programowaniu jest nazewnictwo, w sumie jak zaczynałem to nie wiedziałem jak ważne.

w tym przypadku `setModalState` niewiele mówi co ustawiamy, dlatego podmieniłbym to na `setModalOpen`

będzie jasne że kontrolujemy tym otwieranie i zamykanie modala